Examples/Algorithms/Geant4/src/Geant4Options.cpp

```diff
@@ -78,8 +76,8 @@ ActsExamples::Options::readGeantinoRecordingConfig(
   gRecConfig.generationConfig.vertexSigmaZ =
       variables["g4-vertex-sigmaZ"].as<double>();
 
-  read_range iphir = variables["g4-phi-range"].template as<read_range>();
-  read_range ietar = variables["g4-eta-range"].template as<read_range>();
+  auto iphir = variables["g4-phi-range"].template as<Doubles<2>>();
+  auto ietar = variables["g4-eta-range"].template as<Doubles<2>>();
```

Comment:
nitpick are we really wanna say these are doubles, or just `Reals`

Reply:
You are right, we do not really care that they are doubles. `Reals` might be a good alternative.

Reply:
Renamed to `Reals<...>` and `VariableReals`.